crates/sui-indexer-alt/src/ingestion/mod.rs

```diff
@@ -30,7 +30,7 @@ mod remote_client;
 mod test_utils;
 
 #[derive(clap::Args, Clone, Debug)]
-pub struct IngestionArgs {
+pub struct ClientArgs {
```

Comment:
What is the motivation on the naming change here? I suppose to clarify that this is the source of checkpoints vs configuration for how its used in Ingestion?

Reply:
Yeah, the fact that we had `IngestionArgs` and `IngestionConfig` was grating at me, but then realising that the `IngestionArgs` were only every used to configure the client I thought that helped justify why one should be provided on the command-line and the other should be provided in a file, so I wanted to draw attention to that.